manifests/pnp4nagios/install.pp

```diff
@@ -0,0 +1,13 @@
+class icinga2::pnp4nagios::install inherits icinga2::pnp4nagios::params {
+
+  package {$package_name:
```

Comment:
here you are assuming the $package_name comes from the params.pp. But what happens if the use change the service name in the pnp4nagios class? will the change in package name gets reflected here?